design-components/packages/railz-components/src/components/railz-button/railz-button.tsx

```diff
@@ -63,11 +63,17 @@ export class RailzButton {
     }
   }
 
+  private openNewTab(event: any, url: string): void {
+    event.preventDefault();
+    const newTab = window.open(url, '_blank', 'noopener,noreferrer');
+    newTab.opener = null;
+  }
+
   render(): HTMLElement {
     return (
       <Host class={`button ${this.buttonStyles()}`}>
         {this.href ? (
-          <a href={this.href} target={this.target} class={`button ${this.buttonStyles()}`}>
+          <a href={this.href} rel="noopener noreferrer" class={`button ${this.buttonStyles()}`} onClick={e => this.openNewTab(e, this.href)}>
```

Comment:
So... I might be wrong but... If you are using JavaScript to open the link, you don't need to put the rel="no.....", Don't you?

Reply:
In the html I mean, in the function it's fine